py-polars/polars/internals/anonymous_scan.py

```diff
@@ -85,7 +85,7 @@ def _scan_ds(
 
 
 def _scan_ipc_impl(  # noqa: D417
-    uri: str, with_columns: list[str] | None, *args: Any
+    uri: str, with_columns: list[str] | None, *args: Any, **kargs: Any
```

Comment:
Can you name them `**kwargs`? That's a more conventional naming.

Reply:
Yep, typo! I definitely should have known that 😅